cdap-ui/app/cdap/components/Wrangler/data-buckets.js

```diff
@@ -57,3 +58,31 @@ export function createBucket(array, columnName, columnType) {
   };
 }
 
+// function _createBooleanDistribution(array, columnName) {
```

Comment:
That is to prepare for making Pie chart for boolean column type instead of Histogram